wagtail/wagtaildocs/api/admin/endpoints.py

```diff
@@ -0,0 +1,7 @@
+from __future__ import absolute_import, unicode_literals
+
+from ..v2.endpoints import DocumentsAPIEndpoint
+
+
+class DocumentsAdminAPIEndpoint(DocumentsAPIEndpoint):
+    pass
```

Comment:
It seems to me that we also need the following fields in the Admin API:
* `created_at` - required for list ordering and for "Uploaded" column on the list view
* `filesize` - required for the detail view

The same fields we need for images.